lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -38,6 +39,7 @@ class RenderForm extends React.Component {
   submitForm() {
     const url = 'https://arbisoft.zendesk.com/api/v2/tickets.json',
       $userInfo = $('.user-info'),
+      $this = this,
```

Comment:
`$this` implies that "this" is a jquery object, but I believe it's react though, right?